tests/unit/components/ObsEdit/ObsEdit.test.js

```diff
@@ -99,6 +104,51 @@ test( "renders observation photo from photo gallery", ( ) => {
   expect( getByText( new RegExp( obs.longitude ) ) ).toBeTruthy( );
 } );
 
+describe( "Fetch User Location", () => {
+  beforeEach( () => {
+    // resets mock back to original state
+    mockFetchUserLocation.mockReset();
+  } );
+
+  test( "fetch location when new observation user hasn't saved", async ( ) => {
```

Comment:
Again, try and name your tests so they work as a sentence in the form of "[describe] [test]" which here would result in "Fetch User Location fetch location when new observation user hasn't saved". This isn't purely for style: it easier for other developers to understand the purpose of the test, both in test output and when reading the code.

So here you could refactor like this:

```javascript
describe( "location fetching", ( ) => {
  it( "should run for a new observation", ( ) => { /* ... */ } );
} ); 
```

that way the output looks like 

```
location fetching
  should run for a new observation
```